test/integration/pageobjects/contentEditor.page.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * ContentEditorPage
+ *
+ * A page object API around the Content Editor page.
+ */
+
+var BaseFederalistPage = require('./baseFederalist.page');
+
+var FEDERALIST_TEST_USER = process.env.FEDERALIST_TEST_USER;
+
+function ContentEditorPage () {
+  BaseFederalistPage.apply(this, arguments);
+
+  this.url = [
+    '/#edit',
+    FEDERALIST_TEST_USER,
+    'microsite-template',
+    'gh-pages'
+  ].join('/');
```

Comment:
Maybe take some of these as arguments? I could see wanting to specify the repo name for example.